index.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+<html lang="en-us">
+  <head>
+    <meta charset="UTF-8" />
+    <link rel="stylesheet" href="./assets/css/style.css" />
+    <title>Horiseon - SEO Experts</title>
+  </head>
+
+  <body>
+    <!--section containing header and nav bar embedded within containing links-->
+    <header class="header">
+      <h1>Hori<span class="seo">seo</span>n</h1>
+      <!-- section containing nav with individual links embedded leading to different parts of the site-->
+      <nav>
+        <ul>
+          <li>
+            <a href="#search-engine-optimizatoin">Search Engine Optimization</a>
+          </li>
+          <li>
+            <a href="#online-reputation-management"
+              >Online Reputation Management</a
+            >
+          </li>
+          <li>
+            <a href="#social-media-marketing">Social Media Marketing</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <div class="hero"></div>
+    <div class="content">
+      <!-- main section starts here-->
+      <section class="search-engine-optimization banner-titles">
+        <img
+          src="./assets/images/search-engine-optimization.jpg"
+          class="float-left"
+        />
+        <h2>Search Engine Optimization</h2>
+        <p>
+          The dominance of mobile internet use means that users are searching
+          for the right business as they travel, shop, or sit on their couch at
+          home. Search Engine Optimization (SEO) allows you to increase your
+          visibility and find the right customers for your business.
+        </p>
+      </section>
+      <section
+        id="online-reputation-management"
+        class="online-reputation-management"
+      >
+        <img
+          src="./assets/images/online-reputation-management.jpg"
+          class="float-right"
+        />
+
+        <h2>Online Reputation Management</h2>
+        <p>
+          The web is full of opinions, and some of these can be negative. Social
+          media allows anyone with an internet connection to say whatever they
+          want about your business. Online Reputation Management gives you the
+          control over what potential customers see when they search for your
+          business.
+        </p>
+      </section>
+
+      <section id="social-media-marketing" class="social-media-marketing">
+        <img
+          src="./assets/images/social-media-marketing.jpg"
+          class="float-left banner-images"
+        />
+        <h2>Social Media Marketing</h2>
+        <p>
+          Social media continues to have a sizable influence on buying habits.
+          Social media marketing helps you determine which platforms are suited
+          to your brand, using analytics to find the right markets and increase
+          your lead generation.
+        </p>
+      </section>
+    </div>
+    <div class="benefits">
+      <div class="benefit">
```

Comment:
The `div` tags from here down could have all been `section` tags. This would of been more semantic.